### PR TITLE
Add 2022-02-24 agenda and minutes

### PR DESCRIPTION
**Pull request by csarven**
Request to merge in 24 hours following meeting.

### CODE REVIEW

**Comment by csarven** [comment]
The meeting took place however we did not scribe and there was no voting/decision/action. The discussion was based on the topics as noted in the agenda.

In the past, we cancelled meetings with less than 5 participants - only held them informally and did not scribe/publish the minutes.

Are there any objections to lifting that unwritten "rule" so that we can have the meeting minuted and leave any decision making requiring votes/actions to meetings with at least 5 participants?

If no objections, we can merge this PR.

We can also consider updating the charter ( https://github.com/solid/process/blob/main/notifications-panel-charter.md#decision-policy ) to make this more explicit if the group would like to. I can add this to next meeting's agenda.

**Comment by TallTed** [comment]
Optimally, any scheduled meeting of 2 or more invited participants should be minuted.  Practically, there might be a larger minimum attendance (3? 4? 5?) for minutes to be produced.  

Certainly there should be some quorum required for significant decisions/actions — and minutes should always be produced for meetings that output such decisions/actions — and there should typically be a one or two week window for objections to the decision/action to be lodged by readers of the minutes of those decisions/actions who were unable to attend the meeting for whatever reason.

In my experience, informal (usually meaning there's little to no agenda for the discussion) meetings can produce a lot of useful progress, and informal minutes (i.e., notes) of these informal sessions can help those who were not in attendance benefit from that progress.

There's my $1.35 (two cents, adjusted for inflation).

File: meetings/2022-02-24.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* name
+
+---
+
+## Topics
```
Comment (acoburn) [inline meetings/2022-02-24.md:36]: Would it make sense to be more explicit that no notes were taken and that this reflects the agenda but not the discussion?

Comment (csarven) [reply meetings/2022-02-24.md:36]: That's perfect, thanks.

File: meetings/2022-02-24.md [hunk 2]
```diff
@@ -35,6 +35,9 @@
 
 ## Topics
 
+This meeting was not scribed. The topics below reflects only the agenda but not the discussion.
```
Comment (TallTed) [inline meetings/2022-02-24.md:38]:
```suggestion
This meeting was not scribed. The topics below reflect only the agenda, not the discussion.
```

Comment (timbl) [reply meetings/2022-02-24.md:38]: You could argue in fact that there is a  value of the minutes which is proportional to the number of people who did NOT attend!

**Comment by csarven** [comment]
Merging this PR as is on the basis that the group agreed to not enforce the unwritten rule on minimum number of participants. See 2022-03-03 minutes ( PR'd https://github.com/solid/notifications-panel/pull/49 ) . The group will act in good faith to communicate the decisions/actions - draw people's attention to it so that others can respond.

**Pull request merged**